test/functional/lua/watch_spec.lua

```diff
@@ -126,6 +126,8 @@ describe('vim._watch', function()
         expected_events = expected_events + 1
         wait_for_events()
 
+        vim.wait(100)
```

Comment:
The test did become considerably more consistent when this was even 1ms, but still saw it fail after a few hundred runs at 10ms.